modules/otel-collector-config-options.adoc

```diff
@@ -0,0 +1,148 @@
+// Module included in the following assemblies:
+//
+// * otel/otel-configuration-of-collector.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="otel-collector-config-options_{context}"]
+= OpenTelemetry Collector configuration options
+
+The OpenTelemetry Collector consists of five types of components that access telemetry data:
+
+Receivers:: A receiver, which can be push or pull based, is how data gets into the Collector. Generally, a receiver accepts data in a specified format, translates it into the internal format, and passes it to processors and exporters defined in the applicable pipelines. By default, no receivers are configured. One or more receivers must be configured. Receivers may support one or more data sources.
+
+Processors:: Optional. Processors process the data between it is received and exported. By default, no processors are enabled. Processors must be enabled for every data source. Not all processors support all data sources. Depending on the data source, multiple processors might be enabled. Note that the order of processors matters.
```

Comment:
Curious, since Processors is optional, should it be last in the order?

Reply:
From what I understand here, the order is logical here (receivers > processors > exporters).